packages/react/src/components/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import type { Meta, StoryObj, StoryFn } from '@storybook/react';
+
+import { Chip } from './index';
+
+type Story = StoryObj<typeof Chip>;
+
+export default {
+  title: 'Kjernekomponenter/Chip',
+  component: Chip,
+  parameters: {
+    status: {
+      type: 'beta',
+      url: 'http://www.url.com/status',
+    },
+  },
+} as Meta;
+
+export const Preview: Story = {
+  args: {
+    children: 'Nynorsk',
+    size: 'small',
+    selected: false,
+    checkmark: false,
+  },
+};
+
+export const Default: StoryFn<typeof Chip> = () => {
+  return <Chip>Nynorsk</Chip>;
+};
+
+export const ChipWithCheckmark: StoryFn<typeof Chip> = () => {
+  return (
+    <Chip
```

Comment:
Ser ut som checkmark-ikonet tar opp en del plass. Vil dette føre til problemer når noe skal endre fra unchecked til checkarmed? Da vil jo komponenten ta opp mer plass. Mitt forslag er å legge på horisontal padding når checkmark=true slik at komponenten har samme størrelse uansett om den er selected eller ikke.

Reply:
I'm unsure about the potential impact of the sizing issue. It could potentially result in some jumping, but I believe it's important to have a discussion with our designer about it when she returns after the weekend. 💯 

![image](https://github.com/digdir/designsystem/assets/46874830/249399c8-9924-4760-ba83-1414cebdde02)

Reply:
Blir samme som på [aksel sin chip-komponent](https://aksel.nav.no/komponenter/core/chips). 

Endringen gjør det ekstra tydelig at den blir markert. Størrelsesendringen bidrar også til dette.